diesel/Cargo.toml

```diff
@@ -24,7 +24,7 @@ include = [
 byteorder = { version = "1.0", optional = true }
 chrono = { version = "0.4.20", optional = true, default-features = false, features = ["clock", "std"] }
 libc = { version = "0.2.0", optional = true }
-libsqlite3-sys = { version = ">=0.17.2, <0.31.0", optional = true, features = ["bundled_bindings"] }
+libsqlite3-sys = { version = ">=0.17.2, <0.31.0", optional = true, features = ["bundled"] }
```

Comment:
I would prefer not to enable the bundled feature here as that breaks anyone that requires dynamically linking libsqlite3.

I would rather prefer to have a `SELECT sqlite_version();` query in the doc tests and just skip the test if the local version is too old.

Reply:
In fact, this has nothing to do with the version of `sqlite3`, but with the feature switch of the `libsqlite3-sys` crate that `diesel` depends on. If we continue to use the `bundled_bindings` feature instead of `bundled`, even if the sqlite3 version is higher than 3.45.0 (mine is 3.47.2), it will still report an error of `no such function: jsonb`. This means that we cannot test `jsonb` at all, and (possibly) all JSONB-related functions mentioned in that issue.

Reply:
I have raised an issue under `rusqlite` (the repository where `libsqlite3-sys` is located) and received a relevant answer. https://github.com/rusqlite/rusqlite/issues/1614

Reply:
I've tested that locally and I believe it really just depends on which libsqlite3 (system library, not rust crate) version is linked. If that version is newer than 3.45 it works, if it's older it doesn't work. At least that's the case with my local 3.46 version.

Reply:
Oh you are right, I rechecked my libsqlite and found that it was linked to the system library provided by MacOS instead of the library provided by homebrew. 😭